rules-reviewed/camel3/camel2/tests/data/xml-moved-components/context.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+git git<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
seems this git git text was not put on the place you wanted ?

Reply:
omg ..